components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/security/APIKeyValidator.java

```diff
@@ -539,15 +545,15 @@ private boolean isResourcePathMatching(String resourceString, ResourceInfoDTO re
     }
 
     @MethodStats
-    private APIInfoDTO doGetAPIInfo(String context, String apiVersion) throws APISecurityException {
-        ArrayList<URITemplate> uriTemplates = getAllURITemplates(context, apiVersion);
+    private APIInfoDTO doGetAPIInfo(MessageContext messageContext, String context, String apiVersion) throws APISecurityException {
+        ArrayList<URITemplate> uriTemplates = getAllURITemplates(messageContext, context, apiVersion);
 
         return mapToAPIInfo(uriTemplates, context, apiVersion);
     }
 
     @MethodStats
-    private APIInfoDTO doGetAPIProductInfo(String context, String apiVersion) throws APISecurityException {
-        ArrayList<URITemplate> uriTemplates = getAPIProductURITemplates(context, apiVersion);
+    private APIInfoDTO doGetAPIProductInfo(MessageContext messageContext, String context, String apiVersion) throws APISecurityException {
+        ArrayList<URITemplate> uriTemplates = getAPIProductURITemplates(messageContext, context, apiVersion);
```

Comment:
same here, seems we are exceeding 120

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/9250